### ISSUE TITLE
[New Concept Exercise] : imports

### ISSUE THREAD

**Issue by DavidGerva** [exercism/python#2289]
This issue describes how to implement the `imports` concept exercise for the Python track, which should explain how & why it is useful to `import` names (libraries, modules, classes, functions, etc.) in Python.  We're naming the concept "imports" to avoid a file name clash with the `import` keyword.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This concept exercise should convey a basic understanding and usage of `import` and `import from` in a Python program.  Additionally, the student should learn how to employ the [as](https://yawpitchroll.com/posts/the-35-words-you-need-to-python/#as) keyword for _**aliasing**_  (re-naming) in an`import` context.

## Learning objectives

- Use the `import` keyword to import an entire Python library or module for use in a program
- use the `as` keyword to refer to an imported module by some other (_aliased_) name
- use the `from` keyword to import only specific classes/functions/names from a library or module
- use the  `as` keyword with the `from` keyword to alias or re-name the specific classes/functions/names imported from a library or module.
- be aware of `import` conventions as outlined in [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) 

## Out of scope

- The `importlib` module, and the customization of import behavior.

## Concepts

- `import` statement
- `import from` statement
- `import ... as` statement

## Prerequisites

- `basics`

## Resources to refer to

- [`import` (python docs)](https://docs.python.org/3/reference/simple_stmts.html#the-import-statement)
- [yawpitchroll blog: **import** (_The 35 words you need to Python_)](https://yawpitchroll.com/posts/the-35-words-you-need-to-python/#import)
- [yawpitchroll blog: **as** (_The 35 words you need to Python_)](https://yawpitchroll.com/posts/the-35-words-you-need-to-python/#as)
- [Real Python: the Import keywords](https://realpython.com/python-keywords/#import-keywords-import-from-as)
- [PEP-8: imports](https://www.python.org/dev/peps/pep-0008/#imports)

### Hints

- Hints should refer to one or more of the links above, or analogous links from trusted sources or the Python docs.

## Concept Description 
(_a variant of this can be used for the `v3/languages/python/concepts/<concept>/about.md` doc and this exercises  `introduction.md` doc._)

_**Concept Description Needs to Be Filled In Here/Written**_

_Some "extras" that we might want to include as notes in the concept description, or as links in `links.json`:_
- Considerations and conventions when importing a large amount of libraries.
- Specific conventions used in large frameworks and projects such as [django](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports) or [pandas](https://stackoverflow.com/questions/35697404/why-its-a-convention-to-import-pandas-as-pd)
- More details on Pythons [import system](https://docs.python.org/3/reference/import.html) and `importlib`
- [Modules](https://docs.python.org/3/tutorial/modules.html) and and the [module search path](https://docs.python.org/3/tutorial/modules.html#the-module-search-path) 

## Representer

No changes required.

## Analyzer

No changes required.

## Implementing

The general Python track concept exercise implantation guide can be found [here](https://github.com/exercism/v3/blob/master/languages/python/reference/implementing-a-concept-exercise.md).

Tests should be written using `unittest.TestCase` and the test file named `imports_test.py`.

Code in the `.meta/example.py` file should **only use syntax & concepts introduced in this exercise or one of its prerequisites.** Please do not use comprehensions, generator expressions, or other syntax not previously covered.  Please also follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue, or contact one of the maintainers on our Slack channel.

## Edits
- Added additional links for `as` and `from` keywords.  Re-phrased goals and objectives for clarity.  @BethanyG

**Comment by BethanyG** [comment]
Not yet sure we need/want an entire exercise devoted to `import`.  Putting this back on hold until we decide.

**Comment by BethanyG** [comment]
Flagging as close and re-open with new issue template.  Also needs clarification on scope.
